spec/index.bs

```diff
@@ -1533,7 +1540,98 @@ dictionary GPUBufferBinding {
+        1. Ensure |bindingDescriptor|.{{GPUBindGroupBinding/resource}} is a
+            valid {{GPUSampler}} object.
+    1. If |layoutBinding|.{{GPUBindGroupLayoutBinding/type}} is
+        {{GPUBindingType/"sampled-texture"}} or {{GPUBindingType/"storage-texture"}}.
+        1. Ensure |bindingDescriptor|.{{GPUBindGroupBinding/resource}} is a
+            valid {{GPUTextureView}} object.
+        1. Ensure [=texture view binding validation=] is not violated.
+    1. If |layoutBinding|.{{GPUBindGroupLayoutBinding/type}} is
+        {{GPUBindingType/"uniform-buffer"}} or {{GPUBindingType/"storage-buffer"}}
+        or {{GPUBindingType/"readonly-storage-buffer"}}.
+        1. Ensure |bindingDescriptor|.{{GPUBindGroupBinding/resource}} is a
+            valid {{GPUBufferBinding}} object.
+        1. Ensure [=buffer binding validation=] is not violated.
+1. Return a new {{GPUBindGroup}} object with:
+    - {{GPUBindGroup/[[layout]]}} = |descriptor|.{{GPUBindGroupDescriptor/layout}}
+    - {{GPUBindGroup/[[bindings]]}} = |descriptor|.{{GPUBindGroupDescriptor/bindings}}
+
+<b>Validation Conditions</b>
+
+<dl dfn-for="createBindGroup(descriptor)">
+If any of the following conditions are violated:
+    1. Generate a {{GPUValidationError}} in the current scope with appropriate error message.
+    1. Create a new [=invalid=] {{GPUBindGroup}} and return the result.
+
+<dfn>bind group device validation</dfn>: The {{GPUDevice}} must not be lost.
+
+<dfn>texture view binding validation</dfn>: Let |view| be
+    |bindingDescriptor|.{{GPUBindGroupBinding/resource}}, a {{GPUTextureView}}.
+    This |layoutBinding| must be compatible with this |view|. This requires:
+    1. Its |layoutBinding|.{{GPUBindGroupLayoutBinding/textureDimension}} must equal |view|'s
+        {{GPUTextureViewDescriptor/dimension}}.
+    1. Its |layoutBinding|.{{GPUBindGroupLayoutBinding/textureComponentType}} must be compatible
+        with |view|'s {{GPUTextureViewDescriptor/format}}.
+    1. If |layoutBinding|.{{GPUBindGroupLayoutBinding/multisampled}} is `true`, |view|'s texture's
+        {{GPUTextureDescriptor/sampleCount}} must be greater than 1.
+        Otherwise, if |bindingDescriptor|.{{GPUBindGroupLayoutBinding/multisampled}} is `false`,
+        |view|'s texture's {{GPUTextureDescriptor/sampleCount}} must be 1.
+    1. If |layoutBinding|.{{GPUBindGroupLayoutBinding/type}} is
+        {{GPUBindingType/"sampled-texture"}}, |view|'s texture's {{GPUTextureDescriptor/usage}}
+        must include {{GPUTextureUsage/SAMPLED}}.
+    1. If |layoutBinding|.{{GPUBindGroupLayoutBinding/type}} is
+        {{GPUBindingType/"storage-texture"}}, |view|'s texture's {{GPUTextureDescriptor/usage}}
+        must include {{GPUTextureUsage/STORAGE}}.
+
+<dfn>buffer binding validation</dfn>: A compatible buffer BindGroupLayout binding must exist
+    in {{GPUBindGroupLayoutDescriptor/bindings}} of |descriptor|.{{GPUBindGroupDescriptor/layout}}.
+    This requires:
```

Comment:
This section needs a similar update. (`Let |bufferBinding| be ...`)

Reply:
oops... Done!